<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>The computer Science history database</string>
  </property>
  <property name="windowIcon">
   <iconset resource="myresources.qrc">
    <normaloff>:/icon/monitor.png</normaloff>:/icon/monitor.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="Scientists">
       <attribute name="title">
        <string>Scientists</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="input_scientist_search">
          <property name="placeholderText">
           <string>Type here to search for a specific scientist...</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="combobox_filter_students"/>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_2"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="table_Students">
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Gender</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Birth year</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Death year</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Nationality</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="button_Add_Scientists">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Add new scientist</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_remove_scientist">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Remove selected</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="button_edit_scientist">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Edit scientist information</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Computers</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLineEdit" name="line_input_Computer_name">
          <property name="placeholderText">
           <string>Type here to search for a specific computer...</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QComboBox" name="comboBox_filte_Computers"/>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_3"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="table_computers">
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Build year</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Type</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Constr</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="button_add_computer">
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_delete_computer">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Remove selected computer</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="button_Edit_Computer">
          <property name="text">
           <string>Edit Computer informations</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Relations">
       <attribute name="title">
        <string>Relations</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QTabWidget" name="tabWidget_2">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <widget class="QWidget" name="Scientists_Computer">
             <attribute name="title">
              <string>Scientists - Computer</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLineEdit" name="input_Scientists_id"/>
              </item>
              <item>
               <widget class="QTableWidget" name="table_Scientists_Computer">
                <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                 <bool>false</bool>
                </attribute>
                <attribute name="horizontalHeaderStretchLastSection">
                 <bool>true</bool>
                </attribute>
                <column>
                 <property name="text">
                  <string>Scientists Name</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Computer Name</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="computer_Scientists">
             <attribute name="title">
              <string>Computer - Scientists</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QLineEdit" name="Input_Computer_id"/>
              </item>
              <item>
               <widget class="QTableWidget" name="table_Computer_Scientists">
                <attribute name="horizontalHeaderStretchLastSection">
                 <bool>true</bool>
                </attribute>
                <column>
                 <property name="text">
                  <string>Computer Name</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Scientists Name</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="viewallconnections">
             <attribute name="title">
              <string>View all relations</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QTableWidget" name="table_View_join_Connections">
                <attribute name="horizontalHeaderStretchLastSection">
                 <bool>true</bool>
                </attribute>
                <column>
                 <property name="text">
                  <string>ID</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Scientists Name</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Computer Name</string>
                 </property>
                </column>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QPushButton" name="button_add_connection">
                  <property name="text">
                   <string>Add relation</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="button_remove_connection">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Remove relation</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_test_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_test">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="myresources.qrc"/>
 </resources>
 <connections/>
</ui>
